src/styles/blocks/stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Використовуй &
```suggestion
&__star {
```
та внеси у .stars